src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -272,6 +272,7 @@ public function applyDefaults(ContainerBuilder $container, $serviceId, array $at
 
         $defaultAddServices = [
             'model_manager' => sprintf('sonata.admin.manager.%s', $managerType),
+            'data_source' => sprintf('sonata.admin.data_source.%s', $managerType),
```

Comment:
What if there is no `sonata.admin.data_source.orm` ? I'm not sure to understand how it works.

Reply:
AFAIK service like `*.orm` or simmilar (like `sonata.user.orm.manager`) are load in config file depends on manage type like `docrine_orm.php` or `doctrine_mangodb.php`.
```php
$defaultAddServices = [
    'data_source' => sprintf('sonata.admin.data_source.%s', $managerType),
    //'data_source' => sprintf('?sonata.admin.data_source.%s', $managerType), // with ? it is nullable
];
```